### PR TITLE
tilt effect

### LINKED ISSUE #956: ✨[Feature Request]: Tilt image
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

To make the image tilt in the cursor direction

### Add ScreenShots

image

After adding tilt effect

image

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#956

## Description

[Please include a brief description of the changes or features added]

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #956 

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/a4c638d4-c5f4-44ad-b78e-1dc7d7469d26

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the changes

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by varshith257** [comment]
Can you upload a video of changes done in this PR?

**Comment by Anjaliavv51** [comment]
@varshith257

**Comment by Anjaliavv51** [comment]
> Can you upload a video of changes done in this PR?

https://github.com/user-attachments/assets/a4c638d4-c5f4-44ad-b78e-1dc7d7469d26

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

File: dex.html [hunk 1]
```diff
@@ -451,7 +455,45 @@ <h4>RAPIDOC Newsletter</h4><br>
...
+    // First, make sure to include the GSAP library in your HTML file:
+
+// Then add this JavaScript code:
+
+document.addEventListener("DOMContentLoaded", function() {
+  const images = document.getElementsByTagName('img');
+
+  Array.from(images).forEach(img => {
+    img.addEventListener("mousemove", function(event) {
+      const rect = img.getBoundingClientRect();
```
Comment (varshith257) [inline dex.html:469]: Can you move this code to utils folder? If not create a new folder and add it. Reusable code makes codebase cleaner

**Review by varshith257 — CHANGES_REQUESTED** [review]

File: dex.html (hunk 1 above)
Comment (Anjaliavv51) [reply dex.html:469]: sure

**Comment by Anjaliavv51** [comment]
image

@varshith257 code is moved to js folder

File: index.html [hunk 2]
```diff
@@ -126,11 +131,41 @@ <h1 style="font-size: 65px;">
...
       </h4>
     </div>
-    <img class="sec-img" src="images/rd.png" alt="" />
+    <img class="sec-img" src="images/rd.png" alt="" id="sec-img"/>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.4/gsap.min.js"></script>
+<script>
+  const img = document.querySelector('.sec-img');
+  
+  document.addEventListener("mousemove", function(event) {
+    const rect = img.getBoundingClientRect();
```
Comment (varshith257) [inline index.html:143]: Here too use tilt.js

**Review by varshith257 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Anjaliavv51) [reply index.html:143]: okay

Comment (Anjaliavv51) [reply index.html]: it is changed now please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Pull request merged**